jmh/build.gradle.kts

```diff
@@ -24,7 +24,7 @@ dependencies {
             project(":RoaringBitmap"),
             "com.google.guava:guava:${deps["guava"]}",
             "com.googlecode.javaewah:JavaEWAH:1.0.8",
-            "it.uniroma3.mat:extendedset:1.3.4",
+            "com.n3twork.druid:extendedset:1.3.4",
```

Comment:
What is the reason for this change? If we want to "upgrade", we could go with org.apache.druid:extendedset:0.18.0 which is far more recent and has been actively maintained.

Reply:
resolved